app/models/subnet/ipv6.rb

```diff
@@ -0,0 +1,52 @@
+require 'socket'
+
+class Subnet::Ipv6 < Subnet
+  def self.default_sti_class
+    'Subnet::Ipv6'
+  end
+
+  has_many :interfaces, :class_name => '::Nic::Base', :foreign_key => :subnet6_id
+  has_many :primary_interfaces, -> { where(:primary => true) }, :class_name => 'Nic::Base', :foreign_key => :subnet6_id
+  has_many :hosts, :through => :interfaces
+  has_many :primary_hosts, :through => :primary_interfaces, :source => :host
```

Comment:
I thought so too, but this did not work. Apparently you have to define ```has_many :interfaces``` before you can use ```has_many :hosts, :through => :interfaces```. Am I doing something wrong?